nextbike/prediction/predict.py

```diff
@@ -0,0 +1,22 @@
+from .. import io
+from sklearn import metrics
+import numpy as np
+
+
+def predict_by_regression(pX_test, py_test):
+    scaler = io.read_object("Standard_Scaler.pkl")
+    model = io.read_object("Linear_Regression_Model.pkl")
+    pca = io.read_object("PCA.pkl")
+    X_test_scaled = scaler.transform(pX_test)
+    X_test_transformed = pca.transform(X_test_scaled)
+    y_predictions = model.predict(X_test_transformed)
+    # TODO: Is it fine to call this method from here or should I return to cli.py and then evaluate metrics?
```

Comment:
Find ich ok hier, da "nur" printing und keine "logik"

Reply:
same for me. But as I criticised exactly this point in the last meeting I wanted to discuss it at this point :dagger: